etc/dnf/aliases.d/command-abbreviations.conf

```diff
@@ -0,0 +1,19 @@
+[aliases]
+dup = distro-sync
+dsync = distro-sync
+dist-upgrade = distro-sync
```

Comment:
Do we really want to support 3 abbreviations for one command? I would prefer only one (this is the part that I dislike in formal PR)

Reply:
`dup` and `dist-upgrade` are compatibility with `zypper` and `apt`. `dsync` is the short form for the dnf command itself.